tensorflow/tools/ci_build/Dockerfile.custom_op_ubuntu_16

```diff
@@ -75,7 +75,7 @@ RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.6 0
 # Install given tensorflow or tf-nightly version, if not specified, install the # latest official release
 ARG TF_PACKAGE=tensorflow
 ARG TF_PACKAGE_VERSION=
-RUN pip install ${TF_PACKAGE}${TF_PACKAGE_VERSION:+==${TF_PACKAGE_VERSION}}
+RUN pip install ${TF_PACKAGE}-cpu${TF_PACKAGE_VERSION:+==${TF_PACKAGE_VERSION}}
```

Comment:
I'm sending another change that does s/pip/pip3. Do you mind also updating here to avoid conflict? Thanks!

Reply:
Done